data/navneet17360.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Navneet Anand",
+	"type": "community",
+	"bio": "Front-End Developer",
+	"links": [{
+			"name": "Follow me on Instagram",
+			"url": "https://www.instagram.com/navneet_anand_17",
+			"icon": "FaInstagram"
+		},
+		{
+			"name": "Follow me on Github",
+			"url":"https://github.com/navneet17360",
+			"icon:"FaGithub"
```

Comment:
The JSON is not properly formatted, 
You missed to close the quotation for "icon".

```json
"icon": "FaGithb"
```